diff --git a/browser/base/jar.mn b/browser/base/jar.mn
index 783c67d397a541a0e78374dc93c930fbd8d2e7e7..f0b45efab798e0cad39ed21ab3581205d53f238f 100644
--- a/browser/base/jar.mn
+++ b/browser/base/jar.mn
@@ -47,6 +47,22 @@ browser.jar:
         content/browser/browser-fullScreenAndPointerLock.js (content/browser-fullScreenAndPointerLock.js)
         content/browser/browser-gestureSupport.js           (content/browser-gestureSupport.js)
         content/browser/browser-init.js                     (content/browser-init.js)
+        content/browser/vantara/vantara-shield.js               (content/vantara/vantara-shield.js)
+        content/browser/vantara/vantara-protection.js           (content/vantara/vantara-protection.js)
+        content/browser/vantara/vantara-motion.js               (content/vantara/vantara-motion.js)
+        content/browser/vantara/newtab/index.html               (content/vantara/newtab/index.html)
+        content/browser/vantara/newtab/newtab.css               (content/vantara/newtab/newtab.css)
+        content/browser/vantara/newtab/newtab.js                (content/vantara/newtab/newtab.js)
+        content/browser/vantara/newtab/url-parse.js             (content/vantara/newtab/url-parse.js)
+        content/browser/vantara/newtab/tokens.css               (content/vantara/newtab/tokens.css)
+        content/browser/vantara/newtab/animations.css           (content/vantara/newtab/animations.css)
+        content/browser/vantara/newtab/mark.png                 (content/vantara/newtab/mark.png)
+        content/browser/vantara/vantara-search.js               (content/vantara/vantara-search.js)
+        content/browser/vantara/newtab/strings.js               (content/vantara/newtab/strings.js)
+        content/browser/vantara/vantara-leaks.js                (content/vantara/vantara-leaks.js)
+        content/browser/vantara/vantara-theme.js                (content/vantara/vantara-theme.js)
+        content/browser/vantara/modules/VantaraNewTab.sys.mjs   (content/vantara/modules/VantaraNewTab.sys.mjs)
+        content/browser/vantara/vantara-groups.js               (content/vantara/vantara-groups.js)
         content/browser/browser-main.js                     (content/browser-main.js)
         content/browser/browser-menubar.js                  (content/browser-menubar.js)
         content/browser/browser-pageActions.js              (content/browser-pageActions.js)
