diff --git a/browser/actors/AboutNewTabParent.sys.mjs b/browser/actors/AboutNewTabParent.sys.mjs
index 1b0427ef7444efa08c1ba18b207cf2285ae98f07..02dfe177f5875f2d4ccaed4c74691a657e788536 100644
--- a/browser/actors/AboutNewTabParent.sys.mjs
+++ b/browser/actors/AboutNewTabParent.sys.mjs
@@ -63,6 +63,9 @@ export class AboutNewTabParent extends JSWindowActorParent {
 
   async receiveMessage(message) {
     switch (message.name) {
+      case "VantaraNewTab:ResetBlocked":
+        Services.prefs.clearUserPref("vantara.shield.blockedTotal");
+        break;
       case "AboutNewTabVisible":
         {
           const browsingContext = this.browsingContext;
