diff --git a/browser/moz.build b/browser/moz.build
index 5a03cfc6d3f8fe3a677d4d9e9894e69a98d0c01f..a3d367b9b7aa23a152683c9a98a59bdcf09bbf4a 100644
--- a/browser/moz.build
+++ b/browser/moz.build
@@ -56,6 +56,11 @@ JS_PREFERENCE_PP_FILES += [
 ]
 FINAL_TARGET_FILES.defaults += ["app/permissions"]
 
+# Vantara: заводские настройки. Без препроцессора — см. tools/sync-ui.py.
+JS_PREFERENCE_FILES += [
+    "app/profile/00-vantara.js",
+]
+
 with Files("**"):
     BUG_COMPONENT = ("Firefox", "General")
     SCHEDULES.exclusive = ["linux", "macosx", "windows", "firefox"]
